Cargo.toml

```diff
@@ -16,7 +16,7 @@ documentation = "https://docs.rs/hassle-rs"
 [dependencies]
 bitflags = "2"
 com = { version = "0.6", features = ["production"] }
-libloading = "0.7"
+libloading = "0.8"
```

Comment:
```suggestion
# libloading 0.8 switches from `winapi` to `windows-sys`; permit either
libloading = { version = ">=0.7,<0.9", optional = true }
```

`hassle-rs` might want to do something like this, which is something `wgpu` would require when depending on this library (I assume, I don't actually speak for them).

See https://github.com/gfx-rs/d3d12-rs/pull/43 or https://github.com/gfx-rs/wgpu/pull/3711.